### ISSUE TITLE
npx create-nx-workspace with custom preset is blocked by invalid choice error

### ISSUE THREAD

**Issue by duro** [nrwl/nx#9858]
## Current Behavior
Using a custom plugin I built, which has a preset generator, I am unable to run `npx create-nx-workspace` using it, as I am presented an error

## Expected Behavior
The expected behavior is I should be able to run the following command, and it works:

```
npx create-nx-workspace@latest test-workspace --preset=@pineappledev/express
```

I got to this point following the guides on the Nx website, and this tutorial: https://www.youtube.com/watch?v=yGUrF0-uqaU&t=1s

## Steps to Reproduce
The plugin mentioned above has been pushed to the public NPM registry, so this can reproduced by simply running:

```
npx create-nx-workspace@latest test-workspace --preset=@pineappledev/express
```

If it is sucessful, you should end up with a default "apps" style workspace, with a `.vscode/settings.json` file and a slightly modified `.prettierrc`

### Failure Logs

```
$ npx create-nx-workspace@latest pineapple-test --preset=@pineappledev/express                                                                                                                                                                       [12:19:51]
create-nx-workspace [name] [options]

Create a new Nx workspace

Invalid values:
  Argument: preset, Given: "@pineappledev/express", Choices: "apps", "empty", "core", "npm", "ts", "web-components", "angular", "angular-nest", "react", "react-express", "react-native", "next", "nest", "express"
```

### Environment
Since this is using the `npx create-nx-workspace@latest` command, it is using all the latest versions. As well, the plugin was generated using `npx create-nx-plugin@latest`

**Comment by duro** [comment]
Just a hunch, but I feel like this line is hard blocking the ability for anything other than the predefined presets to pass through:

https://github.com/nrwl/nx/blob/master/packages/create-nx-workspace/bin/create-nx-workspace.ts#L136

**Comment by duro** [comment]
Verified my hunch by successfully executing this using 10.9.7:

```
npx create-nx-workspace@13.9.7 test-workspace --preset=@pineappledev/express
```

**Comment by duro** [comment]
I think in this case, we will have to drop the usage of the `choices` modifier, and simply bake the core preset options into the description.

**Comment by duro** [comment]
Please consider the above PR as a fix for this issue.

**Comment by juristr** [comment]
Hey, thanks for reporting this. We just landed a fix for this #9883. We will soon cut a patch release. Thanks again
